opal/core/search/extract.py

```diff
@@ -252,6 +252,42 @@ def __init__(self, model, episode_queryset, user, fields=None):
         )
 
 
+def generate_files(root_dir, episodes, user):
+    """ Generate the files and return a tuple of absolute_file_name, file_name
```

Comment:
Referring to `the files` makes me think we're in a spy drama ;)

Can we call these e.g. `generate_full_extract_files` for more spelling but less likelihood of renaming later ?